src/context/Auth/AuthContext.ts

```diff
@@ -1,8 +1,14 @@
 import React from "react";
+import {
+  LoginHandler,
+  AuthContextValue,
+  Credentials,
+} from "../../helper/Types";
 
 export const AuthContext = React.createContext({
   token: "",
-  login: () => {},
+  login: async ({ username, password }: Credentials) => ({ token: "" }),
```

Comment:
this can just be ReturnType<typeof function_here>['login']

Reply:
Just a placeholder function and in fact does not currently get replaced.
In fact the useAuthService and useAuthContext need to be reworked anyway to get this context to do anything at all.